### PR TITLE
Updated links to current versions

### PR DESCRIPTION
**Pull request by cooperl09**

### CODE REVIEW

**Comment by jamesaoverton** [comment]
The ontology links are good.

Do you have a new tracker that we can use to replace the old tracker PURL? PURLs should be permanent, so I'd prefer to replace the `/tracker` entry rather than remove it.

**Comment by cooperl09** [comment]
Hi James,
I added the link to the tracker back- with the updated one at GH
L

File: config/eo.yml [hunk 1]
```diff
@@ -4,12 +4,13 @@ idspace: EO
...
+- eo.OBO: https://raw.githubusercontent.com/Planteome/plant-environment-ontology/master/plant-environment-ontology.obo
 
 term_browser: ontobee
 example_terms:
 - EO_0007359
 
 entries:
 - exact: /tracker
-  replacement: https://sourceforge.net/tracker/?atid=5010796&group_id=76834&func=browse
+- replacement: https://github.com/Planteome/plant-environment-ontology/issues
```
Comment (jamesaoverton) [inline config/eo.yml]: This line should start with two spaces, not a hyphen and a space.

Sorry to nitpick, but computers are *very* literal-minded.

Comment (cooperl09) [inline config/eo.yml]: Ok, will change, but I just copied and pasted what was in the file previously, then replaced the link to the tracker....

**Pull request merged**